packages/venia-concept/src/components/Checkout/shippingForm.js

```diff
@@ -42,7 +42,10 @@ class ShippingForm extends Component {
 
         if (availableShippingMethods.length) {
             selectableShippingMethods = availableShippingMethods.map(
-                ({ code, title }) => ({ label: title, value: code })
+                ({ carrier_code, carrier_title }) => ({
```

Comment:
This was a bug. We probably didn't notice because we weren't doing anything with this in reality, but the data would have been wrong.